<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task3</class>
 <widget class="QWidget" name="Task3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Тригонометрия</string>
   </property>
  </widget>
  <widget class="QLabel" name="problemLabel1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="problemLabel2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="problemLabel3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="problemLabel4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerLineEdit1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerLineEdit2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerLineEdit3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerLineEdit4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Вернуться</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
